tools/l10n/check_locales.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#  This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""Check imported locales with release locales."""
+
+from os import path, listdir
+from sys import exit
+from locales import RELEASE_LOCALES
+
+PATH = path.join(path.dirname(path.abspath(__file__)), '../../app/src/main/res/')
+IGNORED_DIRECTORIES = ['sw600dp']
+
+def check_locales():
+    print("Checking for imported locales...")
+    got_error = False
+    imported_locales = [
+        directory[7:].replace('-r', '-') for directory in listdir(PATH) if directory.startswith('values-')
+    ]
+    for ignored in IGNORED_DIRECTORIES:
+        try:
+            imported_locales.remove(ignored)
```

Comment:
nit: fwiw, if imported_locales and IGNORED_DIRECTORIES were sets, I think you could simplify this with `imported_locales -= IGNORED_DIRECTORIES`.

Reply:
`IGNORED_DIRECTORIES` is a list of ignored directories starting with `values-` as locales. Currently we are only aware of 1 directory, but list can be extended in future. These are default directories for Android, and I'm not sure if they will be same always and not generated while building application. `try` block was used to avoid unexpected issues with these directories. We just need to have a list of currently imported locales in project. If anyone with Android development experience can give a good info for possible folders with `values-` in their name, that will help a lot.